app/src/pages/inside/stepPage/modals/linkIssueModal/linkIssueModal.jsx

```diff
@@ -138,16 +136,23 @@ export class LinkIssueModal extends Component {
           type: NOTIFICATION_TYPES.ERROR,
         });
       });
-    this.closeModal();
+    this.props.hideModalAction();
```

Comment:
Does it work with `closeConfirmation` correctly?

Reply:
It worked correctly, but had some problems with the `close` and `submit` by the keys of the keyboard and with css transition.
At the moment I decided to rewrite some logic of modal's submit action. When we click `ok` or `submit`, we don't have to handle the confirmation logic, so I created another method (specifically for `submit` actions) that just closes the modal without any confirming actions.
This approach help to work modals correctly according to our backbone version and doesn't broke anything.